app/src/components/LoginForm.js

```diff
@@ -1,55 +1,106 @@
 import React from 'react';
-import TextField from '@material-ui/core/TextField';
-import FormControl from '@material-ui/core/FormControl';
+import { withStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
 import { compose, withHandlers, withState } from 'recompose';
+import FormControl from './FormControl';
+import Notice from './Notice';
+import errorForField from '../utils/errorForField';
+import validateLogin from '../validators/validateLogin';
 
-const handleLogin = ({ email, loginMutation, password }) => () => (
-  loginMutation({ email, password })
-);
+const styles = theme => ({
+  button: {
+    float: 'right',
+    marginTop: theme.spacing.unit
+  },
+  formLoading: {
+    visibility: 'hidden'
```

Comment:
prevent login form from flashing when logged in users come back to the app while the request to check auth is loading